src/lib/biokepi_pipeline.ml

```diff
@@ -202,6 +207,11 @@ let rec to_json: type a. a t -> json =
           "Configuration", Strelka.Configuration.to_json config;
           "input", to_json bam_pair;
         ]]
+    | Virmid (config, bam_pair) ->
```

Comment:
The idea is to build values of the type: `Yojson.Basic.json`: http://mjambon.com/yojson-doc/Yojson.Basic.html